google/cloud/vision_v1p1beta1/proto/image_annotator.proto

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 Google Inc.
+// Copyright 2019 Google LLC.
```

Comment:
This probably should stay at 2017, right? Also, you'll have to take away the period after LLC for license bot to be happy